microsetta_interface/templates/embedded_pdf.jinja2

```diff
@@ -0,0 +1,27 @@
+{% extends "sitebase.jinja2" %}
+{% set page_title = page_title %}
+
+{% block head %}
+
+<style>
+html, body, embed {
+    margin: 0;
+    height: 100%;
+}
+.content {
+    margin: 0;
+    height: 100%;
+}
+#app {
+    margin: 0;
+    height: 100%;
+}
+</style>
+{% endblock %}
+{% block content %}
+<!-- TODO: Needs an animated loading bar -->
+<div class="alert alert-info" role="alert">
+This report is generated by a third party.  Any identifying personal information: age, height, weight, etc was not shared in the generation of this report.
```

Comment:
Can this be: "This report is generated by a third party without information regarding age, height or weight."